code/enums/eDoorId.mdx

```diff
@@ -0,0 +1,18 @@
+```plaintext
```

Comment:
This shouldn't be here. These files will be automatically read like they're \`\`\`c \`\`\` codeblocks